src/client/src/mockData/leftSidebarData.ts

```diff
@@ -1,10 +1,10 @@
-const leftSidebarData: string[] = [
+const TopNavBarData: string[] = [
   "1. Name and Output",
   "2. Project Type",
   "3. Frameworks",
   "4. Pages",
-  "5. Services (Optional)",
+  "5. Add Optional Cloud Services",
   "6. Summary"
 ];
 
-export default leftSidebarData;
+export default TopNavBarData;
```

Comment:
i think we can rename this file to TopNavBarData

Reply:
oh my bad - good catch!